content/docs/command-reference/fetch.md

```diff
@@ -9,6 +9,7 @@ Download files or directories from [remote storage] to the <abbr>cache</abbr>.
 ```usage
 usage: dvc fetch [-h] [-q | -v] [-j <number>] [-r <name>] [-a] [-T]
                  [--all-commits] [-d] [-R] [--run-cache]
+                 [--max-size MAX-SIZE] [--type {metrics,plots}]
```

Comment:
MAX-SIZE -> <bytes> (as below) ... and it's better in general

Reply:
Adjusted.